<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_DB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Database options</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_user"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_password"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_hostDB"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_DBname"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_portDB"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_hostDB">
       <property name="text">
        <string>Host adress:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_DBname">
       <property name="text">
        <string>Database name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_username">
       <property name="text">
        <string>User name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_portDB">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Transport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Transport options</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Transport:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_transport"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Net">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>Network options</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_hostServer"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_hostGateway"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_portGateway"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_hostPort"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Host adress:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Host adress:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>Gateway options</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>Server options</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Timer">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Timer options</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_delay"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Timer delay:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>196</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
